### PR TITLE
docs: cleanup for getting started ul

### PR DESCRIPTION
**Pull request by harrysolovay**

### CODE REVIEW

File: docs/start/start.md [hunk 1]
```diff
@@ -5,12 +5,12 @@ disableTOC: true
...
-- **Amplify UI components** - UI libraries for React, React Native, Angular, Ionic and Vue.
+The Amplify Framework consists of the following products, which can be used to build fullstack iOS, Android, Web, and React Native apps.
 
-The **Amplify [Console](https://aws.amazon.com/amplify/console/)** is an AWS service that provides a git-based workflow for continuous deployment & hosting of fullstack web apps. Cloud resources created by the Amplify CLI are also visible in the Amplify Console.
+- **Amplify [CLI](~/cli/cli.md)** - Configure all the services needed to power your backend through a simple command line interface.
+- **Amplify [Libraries](~/lib/lib.md)** - Use case-centric client libraries to integrate your app code with a backend using declarative interfaces.
+- **Amplify [UI components](~/ui/ui.md)** - UI libraries for React, React Native, Angular, Ionic and Vue.
+- **Amplify [Console](https://aws.amazon.com/amplify/console/)** is an AWS service that provides a git-based workflow for continuous deployment & hosting of fullstack web apps. Cloud resources created by the Amplify CLI are also visible in the Amplify Console.
```
Comment (jpignata) [inline docs/start/start.md]: Should we replace "is an" with "-" to match the bullets about?

Comment (harrysolovay) [reply docs/start/start.md]: Absolutely. Just pushed that change 👍

**Comment by brene** [comment]
Chatted with Harry offline. The language here was intentional after multiple reviews. Abandoning this PR for now.

File: docs/start/start.md [hunk 2]
```diff
@@ -6,11 +6,11 @@ filterKey: integration
...
-- **Amplify [Libraries](../lib)** - Use case-centric client libraries to integrate your app code with a backend using declarative interfaces.
-- **Amplify UI components** - UI libraries for React, React Native, Angular, Ionic and Vue.
 
-The **Amplify [Console](https://aws.amazon.com/amplify/console/)** is an AWS service that provides a git-based workflow for continuous deployment & hosting of fullstack web apps. Cloud resources created by the Amplify CLI are also visible in the Amplify Console.
+- **Amplify [CLI](~/cli/cli.md)** - Configure all the services needed to power your backend through a simple command line interface.
+- **Amplify [Libraries](~/lib/lib.md)** - Use case-centric client libraries to integrate your app code with a backend using declarative interfaces.
+- **Amplify [UI components](~/ui/ui.md)** - UI libraries for React, React Native, Angular, Ionic and Vue.
+- **Amplify [Console](https://aws.amazon.com/amplify/console/)** – An AWS service that provides a git-based workflow for continuous deployment & hosting of fullstack web apps. Cloud resources created by the Amplify CLI are also visible in the Amplify Console.
```
Comment (swaminator) [inline docs/start/start.md:13]: This needs to be in a separate line (kinda) like what it is in prod with a <br/> @brene

**Pull request closed without merging**